Traceback (most recent call last):
  File "../../run_concrete.py", line 106, in <module>
    tmp_flow,tmp_loss=trainRealNVP(
  File "/arc/project/st-tdjc-1/madmix/examples/discrete_toy_examples/twodim/../../../src/concrete.py", line 312, in trainRealNVP
    loss = -flow.log_prob(sample).mean()
  File "/arc/project/st-tdjc-1/madmix/examples/discrete_toy_examples/twodim/../../../src/concrete.py", line 216, in log_prob
    return self.prior.log_prob(z) + logp
  File "/arc/project/st-tdjc-1/madmix/examples/discrete_toy_examples/twodim/../../../src/concrete.py", line 163, in log_prob
    self._validate_sample(value)
  File "/home/giank/.local/lib/python3.8/site-packages/torch/distributions/distribution.py", line 300, in _validate_sample
    raise ValueError(
ValueError: Expected value argument (Tensor of shape (1000, 20)) to be within the support (IndependentConstraint(Real(), 1)) of the distribution ExpRelaxedCategorical(), but found invalid values:
tensor([[ 2.1645e+23, -3.7210e+22, -3.2206e+23,  ...,  7.3597e+24,
          2.0663e+23,  9.2416e+23],
        [ 1.5968e+23, -2.7455e+22, -2.3759e+23,  ...,  5.4295e+24,
          1.5243e+23,  6.8179e+23],
        [ 1.7985e+23, -3.0914e+22, -2.6760e+23,  ...,  6.1152e+24,
          1.7169e+23,  7.6789e+23],
        ...,
        [ 1.1690e+23, -2.0100e+22, -1.7394e+23,  ...,  3.9748e+24,
          1.1159e+23,  4.9912e+23],
        [ 1.1860e+23, -2.0392e+22, -1.7647e+23,  ...,  4.0327e+24,
          1.1322e+23,  5.0639e+23],
        [ 2.1629e+23, -3.7195e+22, -3.2181e+23,  ...,  7.3540e+24,
          2.0645e+23,  9.2345e+23]], grad_fn=<AddBackward0>)
